guides/network-coverage.mdx

```diff
@@ -25,361 +25,82 @@ Particle's Wallet Abstraction solutions support a significant amount of EVM netw
-| Scroll         | ✓                   | `Scroll`, `ScrollSepolia`                                |
-| Merlin         | ✓                   | `Merlin`, `MerlinTestnet`                                |
-| B² Network     | ✓                   | `BSquared`, `BSquaredTestnet`                            |
-| Bitlayer       | ✓                   | `Bitlayer`, `BitlayerTestnet`                            |
-| Xterio         | ✓                   | `XterioBNB`, `XterioETH`, `XterioBNBTestnet`             |
-| Astar zkEVM    | ✓                   | `AstarzkEVMMainet`, `AstarzkEVMTestnet`                  |
-| IoTeX          | ✓                   | `IoTeX`, `IoTeXTestnet`                                  |
-| Immutable      | ✓                   | `ImmutablezkEVMTestnet`                                  |
-| EOSEVM         | x                   | `EOSEVM`, `EOSEVMTestnet`                                |
-| Lorenzo        | ✓                   | `Lorenzo`, `LorenzoTestnet`                              |
-| SKALE          | x                   | `SKALENebula`                                            |
-| zkLink         | x                   | `zkLinkNova`                                             |
-| ThunderCore    | x                   | `ThunderCore`, `ThunderCoreTestnet`                      |
-| Cronos         | x                   | `Cronos`, `CronosTestnet`                                |
-| OKTC           | x                   | `OKTC`, `OKTCTestnet`                                    |
-| Conflux eSpace | x                   | `ConfluxeSpace`, `ConfluxeSpaceTestnet`                  |
-| Viction        | ✓                   | `Viction`, `VictionTestnet`                              |
-| Fuse           | ✓                   | `Fuse`, `FuseTestnet`                                    |
-| Heco           | x                   | `Heco`                                                   |
-| XLayer         | ✓                   | `XLayer`, `XLayerTestnet`                                |
-| Fantom         | ✓                   | `Fantom`, `FantomTestnet`                                |
-| KCC            | x                   | `KCC`, `KCCTestnet`                                      |
-| AINN           | x                   | `AINN`, `AINNTestnet`                                    |
-| TUNA           | x                   | `TUNA`, `TUNATestnet`                                    |
-| ElastOS        | ✓                   | `Elastos`, `ElastosTestnet`                              |
-| 5ire           | ✓                   | `Fiveire`, `FiveireTestnet`                              |
-| Kakarot        | ✓                   | `KakarotTestnet`                                         |
-| Zircuit        | ✓                   | `ZircuitTestnet`                                         |
+| Chains         | Account Abstraction | Variables (`ChainInfo`)                                  | Chain IDs                                 |
```

Comment:
Edited the network coverage page to use the data from the variable pastes directly in the tables. Best not to overwhelm with information / conflicting vars